src/fides/api/alembic/migrations/versions/ed96417b07d8_create_table_for_persisting_.py

```diff
@@ -0,0 +1,48 @@
+"""create table for persisting MonitorExecution records
+
+Revision ID: ed96417b07d8
+Revises: d9237a0c0d5a
+Create Date: 2025-01-27 23:41:59.803285
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import func
+
+# revision identifiers, used by Alembic.
+revision = "ed96417b07d8"
+down_revision = "d9237a0c0d5a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "monitorexecution",
+        sa.Column("id", sa.String(), primary_key=True),
+        sa.Column("monitor_config_key", sa.String(), nullable=False),
```

Comment:
i think it'd be worth indexing on this column, since we're going to be looking up against it quite a bit